Packs/Zoom/Integrations/Zoom/Zoom.py

```diff
@@ -718,8 +756,10 @@ def main():  # pragma: no cover
     params = demisto.params()
     args = demisto.args()
     base_url = params.get('url')
-    api_key = params.get('creds_api_key', {}).get('password') or params.get('apiKey')
-    api_secret = params.get('creds_api_secret', {}).get('password') or params.get('apiSecret')
+    api_key = params.get('creds_api_key', {}).get(
```

Comment:
revert

Reply:
this is a fix @MichaelYochpaz did, it's already morged.